web/codechecker_web/shared/pgpass.py

```diff
@@ -39,11 +39,13 @@ def _match_field(line, field):
         escaped = False
         if not field:
             return None
-        elif field[0] == line[0]:
+
+        if field[0] == line[0]:
```

Comment:
I think we should use elif statement here.

Reply:
Same answer.